File: front/src/app/services/dto/questao-response.dto.ts
import { ItemQuestaoDiaria } from "src/app/model/item-questao-diaria.model";
import { QuestaoDiaria } from "src/app/model/questao-diaria.model";

export interface QuestaoResponseDTO {
    questaoDiaria: QuestaoDiaria;
    itemQuestaoDiariaList: ItemQuestaoDiaria[];
}

File: back/src/main/java/com/projeto2/nexo/rest/dto/QuestaoResponseDTO.java
package com.projeto2.nexo.rest.dto;

import com.projeto2.nexo.entity.ItemQuestaoDiaria;
import com.projeto2.nexo.entity.QuestaoDiaria;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Setter
@Getter
public class QuestaoResponseDTO {
    private QuestaoDiaria questaoDiaria;
    private List<ItemQuestaoDiaria> itemQuestaoDiariaList;
}
